### PR TITLE
Added website page for companies who offer consultancy and enterprise…

### PR DESCRIPTION
**Pull request by kartikaysaxena**
* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Added a new page in website for support and training (#6504)
On clicking the company card, it redirects to the company site in new page.

## Verification

Support-and-Training-Thanos-PR

### CODE REVIEW

**Comment by kartikaysaxena** [comment]
@bwplotka @yeya24 PTAL

**Comment by yeya24** [comment]
Looks pretty awesome, thanks @kartikaysaxena.
One question, it will do nothing when people click `Add your company`?

**Comment by kartikaysaxena** [comment]
> Looks pretty awesome, thanks @kartikaysaxena. One question, it will do nothing when people click `Add your company`?
> 
> I wonder if we want to keep it

Actually it redirects the user to Thanos GitHub repository. Similar to what [Prometheus](https://prometheus.io/support-training/) does

**Review by MichaHoffmann — APPROVED** [review]
that looks pretty good

File: website/data/adopters.yml [hunk 1]
```diff
@@ -237,3 +237,9 @@ adopters:
 - name: TrueLayer
   url: https://truelayer.com/
   logo: truelayer.png
+- name: Cloudraft
+  url: https://cloudraft.io/observability
+  logo: cloudraft-logo.png
+- name: O11y
+  url: https://o11y.eu
+  logo: o11y_logo.jpeg
```
Comment (yeya24) [inline website/data/adopters.yml:245]: Umm do we want to add these 2 companies to adopters?

**Comment by kartikaysaxena** [comment]
@yeya24 PTAL

**Review by MichaHoffmann — COMMENTED** [review]
docs action fails because of formatting issues, otherwise LGTM

**Comment by kartikaysaxena** [comment]
The docs checks describe something about an internal server error, which may not be related to the code of this PR. I may be wrong here, though. Kindly PTAL @MichaHoffmann

**Comment by MichaHoffmann** [comment]
> The docs checks describe something about an internal server error, which may not be related to the code of this PR. I may be wrong here though. Kindly PTAL @MichaHoffmann

Triggered a rerun

File: website/.hugo_build.lock
Comment (MichaHoffmann) [inline website/.hugo_build.lock:1]: Is this accidentally checked in?

Comment (kartikaysaxena) [reply website/.hugo_build.lock:1]: yes, fixed it.

**Comment by kartikaysaxena** [comment]
Docs action is successful now.

**Comment by MichaHoffmann** [comment]
> Docs action is successful now. But I don't know why there is a issue with e2e tests.

They can be somewhat flaky; i reran and it succeeded!

**Review by MichaHoffmann — APPROVED** [review]
lgtm

**Review by yeya24 — APPROVED** [review]

**Comment by bwplotka** [comment]
Thanks!

**Pull request merged**